File: Aula 06/ExemploDiarias.py
qte = int(input("Digite a quantidade de diárias: "))
tipo = input("Digite o tipo de hospedagem desejada (S-D-T)")

if tipo == 's' or tipo == 'S':
    print("Valor a pagar = R$ ", (qte * 255.50))
elif tipo == 'd' or tipo == 'D':
    print("Valor a pagar = R$ ", (qte * 305.50))
elif tipo == 't' or tipo == 'T':
    print("Valor a pagar = R$ ", (qte * 360.50))
else:
    print("Opção inválida")    

File: Aula 04/ExemploDiarias.java
import javax.swing.JOptionPane;

public class ExemploDiarias {
    public static void main(String[] args) {
        int diarias, tipo;

        tipo = Integer.parseInt(JOptionPane.showInputDialog(null, "Digite o tipo de hospedagem:\n" +
                            "1- Simples\n2- Duplo\n3- Triplo"));
                            diarias = Integer.parseInt(JOptionPane.showInputDialog(null, "Digite a quantidade de diárias:"));
        switch(tipo){
            case 1:
                JOptionPane.showMessageDialog(null, "Valor total R$ " + (diarias * 255.50));
                break;
            case 2:
                JOptionPane.showMessageDialog(null, "Valor total R$ " + (diarias * 305.50));
                break; 
            case 3:
                JOptionPane.showMessageDialog(null, "Valor total R$ " + (diarias * 360.50));
                break;
            default:
               JOptionPane.showMessageDialog(null, "Tipo de hospedagem inválida!");
        }
    }   
}
